digital_image_processing/dithering/burkes.py

```diff
@@ -0,0 +1,98 @@
+"""
+Implementation Burke's algorithm (dithering)
+"""
+from cv2 import imread, imshow, waitKey, destroyAllWindows
+import numpy as np
+
+
+class Burkes:
+    """
+    Burke's algorithm is using for converting grayscale image to black and white version
+    Source: Source: https://en.wikipedia.org/wiki/Dither
+
+    Note:
+        * Best results are given with threshold= ~1/2 * max greyscale value.
+        * This implementation get RGB image and converts it to greyscale in runtime.
+    """
+
+    def __init__(self, input_img, threshold: int):
+        self.min_threshold = 0
+        self.max_threshold = int(
+            self.get_greyscale(255, 255, 255)
+        )  # max greyscale value for #FFFFFF
+
+        if not self.min_threshold < threshold < self.max_threshold:
+            raise ValueError(f"Factor value should be from 0 to {self.max_threshold}")
+
+        self.input_img = input_img
+        self.threshold = threshold
+        self.width, self.height = self.input_img.shape[1], self.input_img.shape[0]
+
+        # error table size (+4 columns and +1 row) greater than input image because of lack of if statements
```

Comment:
Please wrap lines to 88 characters max.
```suggestion
        # error table size (+4 columns and +1 row) greater than input image because of
        # lack of if statements
```